lang/obf/en/texts/press-and-blogs.html

```diff
@@ -16,13 +16,13 @@ <h1>Press and blogs</h1>
 Here are the first mentions of Open Beauty Facts:<br>
 
 </li>
-<li><a href="https://www.huffingtonpost.fr/2016/06/23/composition-cosmetiques_n_10629574.html">Un site collaboratif pour connaître la composition de vos cosmétiques</a> -  05/10/2016</li>
-<li><a href="https://linuxfr.org/news/open-beauty-facts-que-contiennent-vraiment-nos-produits-cosmetiques">Open Beauty Facts : que contiennent vraiment nos produits cosmétiques ?</a> -  31/07/2016</li>
-<li><a href="https://twitter.com/Teolemon/status/909453116889817089">Open Beauty Facts, un champion français du crowdsourcing</a>
-<li><a href="https://twitter.com/RatZillaS/status/837251555657076736">Open Beauty Facts - Phosphore</a>
-<li><a href="https://twitter.com/Teolemon/status/776776806565285888">Open Beauty Facts - Linux Pratique</a>
-<li><a href="https://twitter.com/OpenBeautyFacts/status/753319445397921796">Open Beauty Facts - 20 minutes - French edition</a>
-<li><a href="https://www.up-inspirer.fr/26762-produits-de-beaute-a-vos-etiquettes-prets-decryptez">Open Beauty Facts - UP Le Mag</a>
+<li><a href="https://www.huffingtonpost.fr/2016/06/23/composition-cosmetiques_n_10629574.html">A collaborative site to know the composition of your cosmetics</a> (in french) -  05/10/2016</li>
+<li><a href="https://linuxfr.org/news/open-beauty-facts-que-contiennent-vraiment-nos-produits-cosmetiques">Open Beauty Facts : what do our cosmetics really contain?</a> (in french) -  31/07/2016</li>
+<li><a href="https://twitter.com/Teolemon/status/909453116889817089">Open Beauty Facts, a french crowdsource champion</a> (in french)
+<li><a href="https://twitter.com/RatZillaS/status/837251555657076736">Open Beauty Facts - Phosphore</a> (in french)
+<li><a href="https://twitter.com/Teolemon/status/776776806565285888">Open Beauty Facts - Practical Linux</a> (in french)
```

Comment:
Linux Pratique is a magazine name, I wouldn't translate it.

Reply:
OK